### ISSUE TITLE
Latest Gem release does not work with Rails 6.1.0 (though current master does)

### ISSUE THREAD

**Issue by rst** [bokmann/font-awesome-rails#210]
As of right now, the latest version of font-awesome-rails on rubygems.org is v4.7.0.5, from 2019 -- which is missing the more recent changes (present in Github, but not in the release) to make the gem as compatible with Rails 6.1, and update the `.gemspec` accordingly.

So, this release is marked as incompatible with Rails 6.1 -- but unfortunately there's a prior gem release, version 3.1.1.0, whose `.gemspec` specifies no upper bound.  The upshot is that when a Rails app with `gem font-awesome-rails` is upgraded to Rails 6.1.0, bundler silently *downgrades* font-awesome-rails to version 3.1.1.0 -- which, despite what it says in the metadata, doesn't actually work with 6.1.0, resulting in various interesting complaints about undefined CSS classes and so forth.

There's an obvious workaround -- alter the app's `Gemfile` to pull in the gem from this repo, or a local clone, but it would be better all around for there to be an official release which is up to date.

Thanks!

**Comment by ShockwaveNN** [comment]
Duplicate of #206

**Comment by jasonfb** [comment]
since this is left open, I will mark here, the upgrade to Rails 6.1 does appear to not kick-in the mechanism to force update the gem. (I'd recommend you pull the one that allows 6.1 and re-release one for 6.0 with a specific version lock on rails =<6.0). 

the forthcoming `4.7.0.6` does indeed fix this issue— I'm sure it will be pushed to rubygems soon. In the meantime, Rails 6.1 upgraders can:

```
gem 'font-awesome-rails', branch: 'master', github: 'bokmann/font-awesome-rails'
```

**Comment by rmm5t** [comment]
v4.7.0.6 released. Sorry for the delay. Cheers. 🍻

**Comment by jasonfb** [comment]
@rmm5t -- you still might want to yank 3.1.1.0 cause everyone who installs `rails new` will get stuck on that version

**Comment by rst** [comment]
A new 6.1.0 app with `gem 'font-awesome-rails` now gets the (just released) 4.7.0.6.  So, I think we're good until the release of 6.2.0 (or 7.0 if things go that way)...

**Comment by stephenhuey** [comment]
Font Awesome icons are not being loaded when using the Bulma CSS framework although [Bulma expects it to work](https://bulma.io/documentation/elements/icon/).  My app is running on Ruby 3 with Rails `6.1.3.1` and this is in my Gemfile:

```

gem "bulma-rails", "~> 0.9.1"
gem "font-awesome-rails", "4.7.0.6"

```

I initially tried `font-awesome-rails` with no version (the latest version).  Here is my application.scss file:

```

@import "bulma";
@import "font-awesome";

```

Rubymine says _Cannot resolve file 'font-awesome'_ and I'm really not sure of the cleanest way to get icons showing up in my browser.  

Any ideas?
